training/imagenet/README.md

```diff
@@ -0,0 +1,98 @@
+# ImageNet training with DeepSpeed
+
+This example activated DeepSpeed on the implementation of training a set of popular model architectures on ImageNet dataset. The models include ResNet, AlexNet, and VGG, and the
+baseline implementation could be found at pytorch examples [Github repository](https://github.com/pytorch/examples/tree/main/imagenet). DeepSpeed activation allows for ease in
+running the code in distributed manner, allowing for easily applying fp16 quantization benefitting Zero stage1 memory reduction.
+
+## DeepSpeed Optimizations
+
+Applying fp16 quantization and Zero stage 1 memory optimization we were able to reduce the required memory. The table bellow summarizes the results of running resnet 50 on one
+node 16 V100 GPUs:
+
+Optimization level | Allocated Memory (GB) | Improvement (%)
+-------------------|-------------------
+Baseline| ? | -
+Baseline with DS activated | 1.66 | -
+DS + fp16 | 1.04 | ?
+Ds + fp16 + Zero 1 | 0.81 | ?
```

Comment:
table format is not correct. take a look at rendered website

Reply:
Fixed it